Mage.Sets/src/mage/cards/t/ThaliaAndTheGitrogMonster.java

```diff
@@ -0,0 +1,87 @@
+package mage.cards.t;
+
+import mage.MageInt;
+import mage.abilities.common.AttacksTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.effects.common.DrawCardSourceControllerEffect;
+import mage.abilities.effects.common.PermanentsEnterBattlefieldTappedEffect;
+import mage.abilities.effects.common.SacrificeTargetEffect;
+import mage.abilities.effects.common.continuous.PlayAdditionalLandsControllerEffect;
+import mage.abilities.keyword.DeathtouchAbility;
+import mage.abilities.keyword.FirstStrikeAbility;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.FilterPermanent;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterControlledPermanent;
+import mage.filter.predicate.Predicate;
+import mage.filter.predicate.Predicates;
+import mage.target.common.TargetControlledPermanent;
+
+import java.util.UUID;
+
+public class ThaliaAndTheGitrogMonster extends CardImpl {
+
+    private static final FilterPermanent filterTapped = new FilterPermanent("creatures and nonbasic lands your opponents control");
+    private static final FilterControlledPermanent filterAttack = new FilterControlledPermanent("creature or land");
+
+    static {
+        filterAttack.add(Predicates.or(
+                CardType.CREATURE.getPredicate(),
+                CardType.LAND.getPredicate()
+        ));
+        filterTapped.add(Predicates.or(
+                CardType.CREATURE.getPredicate(),
+                CardType.LAND.getPredicate()
+        ));
+        filterTapped.add(Predicates.not(
+                SuperType.BASIC.getPredicate()
+        ));
```

Comment:
this is really nitpicky but the nonbasic predicate should be included with the creature predicate in case there's ever somehow a permanent which is basic but isn't a land or creature